slices/dbus.yaml

```diff
@@ -0,0 +1,34 @@
+package: dbus
+
+essential:
+  - dbus_copyright
+
+slices:
+  bins:
+    essential:
+      - dbus-bin_bins
+      - dbus-daemon_bins
+      - dbus-system-bus-common_config
+      - dbus_config
+      - libc6_libs
+      - libdbus-1-3_libs
+      - libexpat1_libs
+      - libsystemd0_libs
+    contents:
+      /etc/init.d/dbus:
+      /usr/lib/dbus-1.0/dbus-daemon-launch-helper:
+
+  config:
+    contents:
+      /etc/default/dbus:
+
+  extras:
```

Comment:
hum, maybe `services` instead of `extras` to be more representative of what's being installed

Reply:
It's also more consistent with previous packages actually, updating this!